### PR TITLE
doc : Update documentation after NCS 1.3.1

### PR DESCRIPTION
**Pull request by umapraseeda**
Update documentation after NCS 1.3.1 release

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: doc/versions.json [hunk 1]
```diff
@@ -16,6 +17,12 @@
       "zephyr": "2.3.0-rc1",
       "mcuboot": "1.6.0-rc2"
     },
+	"1.3.1": {
+      "ncs": "1.3.1",
+      "nrfxlib": "1.3.1",
+      "zephyr": "2.3.0-rc1",
+      "mcuboot": "1.6.0-rc2"
```
Comment (utzig) [inline doc/versions.json:24]: Is it the correct mcuboot version though?

File: doc/versions.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "VERSIONS": [
     "latest",
+	"1.3.1",
```
Comment (utzig) [inline doc/versions.json:4]: better use spaces instead of tabs here

File: doc/versions.json [hunk 3]
```diff
@@ -16,6 +17,12 @@
       "zephyr": "2.3.0-rc1",
       "mcuboot": "1.6.0-rc2"
     },
+	"1.3.1": {
```
Comment (utzig) [inline doc/versions.json:20]: same comment about spaces vs tabs

File: doc/versions.json [hunk 4]
```diff
@@ -16,6 +17,12 @@
...
+      "ncs": "1.3.1",
+      "nrfxlib": "1.3.1",
+      "zephyr": "2.3.0-rc1",
+      "mcuboot": "1.6.0-rc2"
+	},
```
Comment (utzig) [inline doc/versions.json:25]: same comment about spaces vs tabs

File: doc/versions.json (hunk 1 above)
Comment (ru-fu) [reply doc/versions.json:24]: That was the mcuboot version for 1.3.0, and mcuboot wasn't updated for 1.3.1 - correct @shantha-14 ?

Comment (ru-fu) [reply doc/versions.json:24]: Shantha confirms there is no change from 1.3.0.

File: doc/versions.json (hunk 2 above)
Comment (umapraseeda) [reply doc/versions.json]: Fixed

**Review by utzig — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]
Looks good!
1.3.1 isn't uploaded yet, but we can merge once it is.

**Review by carlescufi — APPROVED** [review]

**Review by nvlsianpu — APPROVED** [review]

**Pull request merged**